src/diagram

```diff
@@ -0,0 +1,9 @@
+ MusicPlaylistApp (Root)
```

Comment:
This diagram doesn't show the component layering. Show how the kernel extends from Standard, etc.